index.html

```diff
@@ -709,6 +709,71 @@ <h2>Exposing MediaStreamTrack source background blur support</h2>
 };</pre>
     </div>
   </section>
+  <section>
+    <h2>Exposing MediaStreamTrack source automatic face framing support</h2>
+    <p>Some platforms or User Agents may provide built-in support for automatic
+       continuous framing based on the position of human faces within the field
+       of view, in particular for camera video streams.
+       Web applications may either want to control or at least be aware that
+       automatic continuous human face framing is applied at the source level.
+       This may for instance allow the web application to update its UI or to
+       not apply human face framing on its own.
+       For that reason, we extend {{MediaStreamTrack}} with the following
+       properties.
+    </p>
+    <p>The WebIDL changes are the following:</p>
+    <pre class="idl"
+>partial dictionary MediaTrackSupportedConstraints {
+  boolean faceFraming = true;
+};
+
+partial dictionary MediaTrackCapabilities {
+  sequence&lt;boolean&gt; faceFraming;
+};
+
+partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean faceFraming;
+};
+
+partial dictionary MediaTrackSettings {
+  boolean faceFraming;
+};</pre>
+    <section>
+      <h3>Processing considerations</h3>
+      <p>When the "faceFraming" setting is set to <code>true</code> by
+         the <a>ApplyConstraints algorithm</a>, the UA will attempt to
+         continuously improve framing by cropping to human faces.
+      </p>
+      <p>When the "faceFraming" setting is set to <code>false</code> by
+         the <a>ApplyConstraints algorithm</a>, the UA will not crop to human
+         faces.
+      </p>
```

Comment:
We need to clarify what happens to width and height of the track, but that can happen in a separate issue